src/Stratis.Bitcoin.Features.PoA/PoAConsensusErrors.cs

```diff
@@ -20,5 +20,9 @@ public static class PoAConsensusErrors
 
         // Collateral related errors.
         public static ConsensusError InvalidCollateralAmount => new ConsensusError("invalid-collateral-amount", "collateral requirement is not fulfilled");
+
+        public static ConsensusError InvalidCollateralAmountNoCommitment => new ConsensusError("invalid-collateral-amount", "collateral requirement is not fulfilled; no commitment found");
+
+        public static ConsensusError InvalidCollateralAmountCommitmentTooNew => new ConsensusError("invalid-collateral-amount", "collateral requirement is not fulfilled; commitment too new");
```

Comment:
Maybe the short description should also differ between these? So that `invalid-collateral-amount` is also unique?

Reply:
Done :)